plugins/modules/win_format.ps1

```diff
@@ -74,7 +74,8 @@ function Get-AnsibleVolume {
 
     if ($null -ne $DriveLetter) {
         try {
-            $volume = Get-Volume -DriveLetter $DriveLetter
+            $partition = Get-Partition -DriveLetter $DriveLetter | Where-Object { $null -ne $_.DiskNumber }
+            $volume = Get-Volume -Partition $partition
```

Comment:
Can you explain how this fixes the issue? When would there be a partition without a DiskNumber and why should we be ignoring them?

Reply:
In scenarios where the failover cluster role is installed on a windows server, Get-Volume and Get-Partition show every disk from every node participating in that cluster. Say we have a clustered drive M. Get-Volume would show you n many entries for drive M. The key differentiator between all those entries being the disk number. Only local disk will ever have a disk number. So with that in mind we can ignore all other disk that do not posses a disk number. The new code makes sure that we will always use the local disk as all relevant calls in the modules only work with local disk.
Hope this helps.

Reply:
Thanks for the explanation, could you possible include a comment that briefly explains this so at least we know not to remove those lines in the future? Same for the other file.

Reply:
@xcom1710 could you add the comment so that this PR can be merged T_T 
Also there is the same issue with win_disk_facts

Reply:
done